generators/express/templates/base/src/index.ts.ejs

```diff
@@ -8,7 +8,7 @@ Sentry.init({
     release: version ? `<%= projectName %>-<%= packageName %>@${version}` : undefined,
 });
 
-const port: number = process.env.PORT ? parseInt(process.env.PORT, 10) : 3000;
+const port: number = process.env.PORT ? parseInt(process.env.PORT, 10) : 4000;
```

Comment:
I would prefer to keep `3000` here to keep in line with what is traditionally done in express projects and always set the `PORT` env variable.